### DISCUSSION TITLE
[@types/react] Deprecate ReactComponentElement

### DISCUSSION THREAD

**Discussion by mattpocock** [DefinitelyTyped/DefinitelyTyped#68457] [Issues with a @types package] [answered]
There are no references to it elsewhere in the React types.

I can't see a reason why it should exist, so we should deprecate it with an eye to removing it.

[a comment by an automated account is not shown]

**Comment by eps1lon** [comment]
Good catch. Can you file a PR deprecating it and then I add it to https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/64451

**Comment by ghost** [comment, marked as answer]
Deprecated in PR #68463
